### PR TITLE
add app kshell

### LINKED ISSUE #70: add k-shell algorithm
Add k-shell algorithm. 
You may find its descriptions here: https://networkx.org/documentation/stable/reference/algorithms/generated/networkx.algorithms.core.k_shell.html#networkx.algorithms.core.k_shell

Please provide its verification (comparing the correctness with networkx) to CI together with the algorithm.

### PR DESCRIPTION
**Pull request by varinic**
## What do these changes do?

Add kshell application to GraphScope analytical_engine.

## Related issue number

Resolves #70.

### CODE REVIEW

**Comment by sighingnow** [comment]
Hi @varinic, thanks for contributing the algorithm. To get this pull request landed, please following our developer guide and error message in CI to fixed the code format of your patch.

**Comment by sighingnow** [comment]
THIMC: pull request https://github.com/7br/gstest/pull/34 needs to be merged first.

**Review by sighingnow — APPROVED** [review]

File: python/graphscope/analytical/app/k_shell.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+        import graphscope as gs
+        s = gs.session()
+        g = s.load_from('The parameters for loading a graph...')
+        pg = g.project_to_simple(v_label='vlabel', e_label='elabel')
+        r = gs.k_shell(pg)
+        s.close()
+
+    """
+    return AppAssets(algo="kshell")(graph, k=k)
```
Comment (lidongze0629) [inline python/graphscope/analytical/app/k_shell.py:51]: Builtin-app doesn't need `kwargs` params.
```python
k = int(k)
return AppAssets(algo="kshell")(graph, k)
```

**Review by lidongze0629 — CHANGES_REQUESTED** [review]

**Comment by yecol** [comment]
please revise according to dongze's comment and make the CI pass.

**Review by yecol — APPROVED** [review]

**Review by lidongze0629 — APPROVED** [review]

**Pull request merged**